src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -491,33 +491,46 @@ public static void TCP_ServiceCertExpired_Throw_MessageSecurityException()
 
     [Fact]
     [OuterLoop]
+    // Confirm that the Validate method of the custom X509CertificateValidator is called and that an exception thrown there is handled correctly.
     public static void TCP_ServiceCertFailedCustomValidate_Throw_Exception()
     {
         string testString = "Hello";
+        NetTcpBinding binding = null;
+        EndpointAddress endpointAddress = null;
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
```

Comment:
We started doing this some time back when we first decided to include the test sections // *** SETUP *** \\, // *** EXECUTE *** \\, etc...

The idea was to divide up the test into its distinct sections the first of which was to instantiate everything and then "setup" and so forth.

When I touch an old test I update it to follow this pattern, eventually we will achieve this consistent pattern across all our tests.